app/components/SelectedAlertShortcutLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link } from '@remix-run/react'
+
+import {
+  JsonNoticeTypes,
+  NoticeTypes,
+} from './NoticeTypeCheckboxes/NoticeTypeCheckboxes'
+import { WithCredentials, useEmail } from '~/root'
+
+export default function SelectedAlertShortcutLink({
+  alertKey,
+  format,
+  otherAlerts,
+  destination,
+}: {
+  alertKey: string
+  format: 'json' | 'text'
+  destination: 'quickstart' | 'email'
+  otherAlerts?: string[] // Alerts specifically under the 'Other' tab, since they can't be generically selected as all
+}) {
```

Comment:
There is actually not much code shared in common among the various `if` statement branches. Would this be cleaner as several different components?

Lots of `if` statements are often a hint that [inheritance or function composition may be a beneficial refactoring](https://refactoring.guru/replace-conditional-with-polymorphism).